apps/rush-lib/src/cli/utilities/ChangelogGenerator.ts

```diff
@@ -43,15 +43,19 @@ export default class ChangelogGenerator {
         // Changelogs should only be generated for publishable projects.
         // Do not update changelog or delete the change files for prerelease.
         // Save them for the official release.
-        if (!!project && project.shouldPublish && !semver.prerelease(project.packageJson.version)) {
+        // Unless the package is a hotfix, in which case do delete the change files.
+        if (!!project && project.shouldPublish &&
+          (!semver.prerelease(project.packageJson.version) ||
+            project.packageJson.version.indexOf('hotfix') >= 0)
```

Comment:
Please do not use a hardcoded condition like this. Are you sure no one else has 'hotfix' as part of their version?